### PR TITLE
Fixing broken tables + Lint Compliance issues

### PR DESCRIPTION
**Pull request by JSzaszvari**
Fixed all the MD022 issues

There was heaps of tables that were not rending at all due to being incorrectly inputted (most of the ones in the chat folder for the rest-api 

as well as a few other lint compliance fixes

### CODE REVIEW

File: developer-guides/rest-api/chat/unstarmessage/README.md [hunk 1]
```diff
@@ -1,26 +1,29 @@
 # Unpins a Chat Message
-Removes the star on the chat message for the authenticated user.
+
+Unpins the chat message for the authenticated user.
 
 | URL | Requires Auth | HTTP Method |
-| :--- | :--- | :--- | :--- |
-| `/api/v1/chat.unStarMessage` | `yes` | `POST` |
+| :--- | :--- | :--- |
+| `/api/v1/chat.unPinMessage` | `yes` | `POST` |
```
Comment (graywolf336) [inline developer-guides/rest-api/chat/unstarmessage/README.md]: This file is named `unstarmessage`, so I think this file is meant to be about the unstaring

**Review by graywolf336 — DISMISSED** [review]

File: developer-guides/rest-api/chat/unstarmessage/README.md (hunk 1 above)
Comment (JSzaszvari) [reply developer-guides/rest-api/chat/unstarmessage/README.md]: Whoops my bad, that's a mistake on my part. I'll fix it now. Apologies

**Comment by JSzaszvari** [comment]
@graywolf336 Just fixed that up now. Sorry mate

**Review by MartinSchoeler — APPROVED** [review]

**Comment by MartinSchoeler** [comment]
Looks good to me. Thanks For the contribution!

**Pull request merged**